stock_card_report/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+Add Stock card to Inventory Reporting.
```

Comment:
Stock Card Report is the report that display movement (in/out) of a product in a specified location and date range.